requirements.txt

```diff
@@ -74,6 +74,7 @@ ruamel.yaml.clib~=0.2.2
 schema~=0.7.4
 Send2Trash~=1.5.0
 six~=1.15.0
+scipy~=1.6.2
```

Comment:
Why Scipy here. QCoDeS does not depend on scipy directly (as stated in setup.cfg) I would prefer to keep this that way since scipy is a somewhat big dependency

Reply:
I used `scipy` to do interpolation, but since it's just interpolation between two datapoints I suppose I could just use `numpy` for that.

Reply:
But that is only in the tests right ?

Reply:
Yes that's right, I've removed it from here but left the dependency in test_requirements.txt.